server/api/ballots.py

```diff
@@ -172,20 +170,11 @@ def serialize_ballot(ballot: SampledBallot) -> JSONDict:
 
 
 def serialize_ballot_comparison_ballot(
-    ballot: SampledBallot, cvr_ballot: CvrBallot, jurisdiction: Jurisdiction
```

Comment:
I thought we wanted to maybe keep the jurisdiction around to filter by jurisdiction... but maybe that's for later?

Reply:
I think what you're referring to is that we only want to show the audit board the contests that meet two conditions:
1. The contest should be selected for audit (either targeted or opportunistic)
2. The jurisdiction should be in the contest universe

We meet (1) because Arlo only stores the contests that are selected for audit. We meet (2) because the /contests endpoint for audit boards scopes the contests returned to only the ones with the jurisdiction in their contest universe.